dev/com.ibm.ws.cdi.api_fat/fat/src/com/ibm/ws/cdi/api/fat/apps/current/CDICurrentTestServlet.java

```diff
@@ -59,7 +59,7 @@ public void testCDICurrentViaMES() throws Exception {
 
         managedExecutorService.submit(new callCDICurrent());
 
-        int i = 10;
+        int i = 40;
```

Comment:
1. so this change equates to going from 50ms to 200ms ... seems like a big step up but actually still 200ms isn't very long. Is that the right amount of time? Do you want to pull the actual sleep time out into a variable as well?
2. counting down rather than up is a bit odd?
3. Should `wasCDICurrentFoundViaMOE` be `wasCDICurrentFoundViaMES`?

Reply:
4. Is `callCDICurrent` a class? Why doesn't it start with a capital?

Reply:
5. Should `wasCDICurrentFoundViaMOE` be either volatile or access synchronized?